go.mod

```diff
@@ -23,7 +24,7 @@ require (
 	golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97
 	golang.org/x/term v0.0.0-20210429154555-c04ba851c2a4
 	golang.org/x/text v0.3.6
-	golang.org/x/tools v0.0.0-20201022035929-9cf592e881e9
+	golang.org/x/tools v0.1.1
```

Comment:
But why?

Reply:
Just `go mod tidy`.

Reply:
I'm afraid it'll break something in the compiler (see #2086).

Reply:
Oh, I got it. Now this module is not touched.